src/Form/Type/ModelAutocompleteType.php

```diff
@@ -119,8 +119,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'to_string_callback' => null,
 
             // add button
-            // NEXT_MAJOR: Set this value to 'link_add' to display button by default
-            'btn_add' => false,
```

Comment:
I'm not sure about this change. The option name is `btn_add`

Isn't something like  `'btn_add' => 'link_add'` or `'btn_add' => options['link_add']`... or anything else. We should look for the PR who added the message.

Reply:
I think I did a mistake here, will take a look on the corresponding PR